tests/sklearn/test_sklearn_autolog.py

```diff
@@ -1462,3 +1462,30 @@ def test_metric_computation_handles_absent_labels():
     assert not metrics
     assert tags == get_expected_class_tags(model)
     assert MODEL_DIR in artifacts
+
+
+def test_autolog_disabled_on_sklearn_cross_val_api():
+    mlflow.sklearn.autolog()
+    from sklearn import linear_model
+    from sklearn.model_selection import cross_validate, cross_val_predict, cross_val_score
+
+    def assert_autolog_disabled(run_):
```

Comment:
Can we rename this function to `assert_run_is_empty` because that's what this function does?